### ISSUE TITLE
favicon.ico uses default image, not the one supplied

### ISSUE THREAD

**Issue by Sujimichi** [RealFaviconGenerator/realfavicongenerator#128]
I tried the site out with the demo image to start with and it looked good so I selected my own image to use.  In the downloaded package all the pngs are correct, but the favicon.ico file shows the demo image. Otherwise this is a nice tool.

**Comment by phbernard** [comment]
Wow, this is bad :(

I can't reproduce the issue:
- I go to realfavicongenerator.net
- I click the green "Demo" button
- I generate the favicon: in the package, favicon.ico contains the demo picture as expected
- I click the Realfavicongenerator logo at the top of the page to go back to the home page.
- I click the blue "Select a picture" button and I pick an image of mine
- I generate the favicon: in the package, favicon.ico contains my picture as expected (not the demo picture).

Is there something you've done differently?

**Comment by Sujimichi** [comment]
Oh...dude, I'm really sorry, I messed up. It works fine. 

What happened was I downloaded the zip after trying it with the demo image and unpacked it into a temp folder and it looked fine. Then I ran it again with my own image and downloaded that zip.  When I unpacked the second zip (into the same folder, replacing the previous images) all the thumbnails showed the image I expected except for the thumnail for the favicon. That showed the thumbnail from the demo zip. (This was in windows 7).
I then went back to Linux and unpacked the zip again and this time all the thumbnails where correct includeing the favicon.  So the file in the zip is fine, it was something to do with windows and caching of thumbnails I suppose. 
Sorry for wasting your time.
